### DISCUSSION TITLE
can i have an option to rotate an object or layer

### DISCUSSION THREAD

**Discussion by Mammad81** [PintaProject/Pinta#597] [Ideas]
can i have an option to rotate an object or layer with degree, something like free transform in photoshop

**Comment by cameronwhite** [comment]
There are a couple ways I can think of:
- the `Rotate / Zoom Layer` option in the `Layers` menu
- In the `Move Selected Pixels` tool, right clicking and dragging allows you to rotate the selected content

Do either of those work for what you're looking for?
